### DISCUSSION TITLE
Accounts and Services

### DISCUSSION THREAD

**Discussion by duncdrum** [eXist-db/exist#4870] [Team Posts]
# Accounts
I think it would be helpful to collect which org members are admins for our various exist related accounts. Don't post credentials just who admins them. 

We should also make sure that there are more than 1 person with access to all of these. 

- [npm](https://www.npmjs.com/search?q=%40existdb)
  - duncdrum
  - line-o
- travis
   - GitHub org admins
- appveyor (Adam's account)
   - adamretter
   - eXist-db/core
- codacy
   - adamretter
- [nexus repo](http://repo.evolvedbinary.com/repository/exist-db/)
  - adamretter
  - wolfgang
- [dockerhub](https://hub.docker.com/u/existdb/dashboard/)
   - wolfgang
   - adamretter
   - joewiz
- [exist-db.org](http://www.exist-db.org)
   - ljo
- Mailing list
   - Wolfgang, Adam, Dannes?
- slack
   - joewiz, adamretter
- [Cypress Dashboard](https://dashboard.cypress.io/organizations/c3fd353f-0298-4ca9-b16a-12096c21dd45/projects)
  - duncdrum

**Comment by joewiz** [comment]
I'm the admin of the slack group. Anyone interested in co-administering?

**Comment by adamretter** [comment]
@joewiz Sure sign me up

**Comment by joewiz** [comment]
@adamretter Ok, you are now a 2nd owner of the eXist-db workspace on Slack.

**Comment by joewiz** [comment]
Wolfgang added me to the eXist organization on Dockerhub, and I have added my Docker credentials to Travis, so our Docker tests that use the HEAD of develop when they run the test suite will pass.
